muted-tests.yml

```diff
@@ -180,6 +180,12 @@ tests:
 - class: "org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT"
   method: "test {scoring.*}"
   issue: https://github.com/elastic/elasticsearch/issues/117641
+- class: "org.elasticsearch.xpack.esql.qa.mixed.MultilusterEsqlSpecIT"
```

Comment:
there is a typo here - also looks like a duplicate with the previous muted test (line 180)

Reply:
oh no! dummy PR to fix that https://github.com/elastic/elasticsearch/pull/118665